WORKSPACE

```diff
@@ -15,16 +15,20 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 versions.check(minimum_bazel_version = "0.23.0")
 
-http_archive(
+git_repository(
     name = "io_bazel_rules_go",
-    sha256 = "6776d68ebb897625dead17ae510eac3d5f6342367327875210df44dbe2aeeb19",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.17.1/rules_go-0.17.1.tar.gz",
+    commit = "0c4a3523fe18441b73308a8d56ea946888aab6bf",
+    remote = "https://github.com/bazelbuild/rules_go.git",
+    shallow_since = "1552603757 -0400",
```

Comment:
neat!

Reply:
something like this could be very useful in the pod-utils

Reply:
Yeah this is a pretty sane way to initially start with a tag and then wait for bazel to tell you how to convert this into its cacheable equivalent.